common/lib/src/realm_types.dart

```diff
@@ -23,17 +23,19 @@ import 'package:sane_uuid/uuid.dart';
 
 /// @nodoc
 abstract class RealmAccessorMarker {
-  T getValue<T>(RealmObjectMarker object, String propertyName);
-  T? getObject<T>(RealmObjectMarker object, String propertyName);
-  List<T> getList<T>(RealmObjectMarker object, String propertyName);
-  void set<T>(RealmObjectMarker object, String propertyName, T value, {bool isDefault = false, bool update = false});
+  T getValue<T>(RealmObjectBaseMarker object, String propertyName);
+  T? getObject<T>(RealmObjectBaseMarker object, String propertyName);
+  List<T> getList<T>(RealmObjectBaseMarker object, String propertyName);
+  void set<T>(RealmObjectBaseMarker object, String propertyName, T value, {bool isDefault = false, bool update = false});
 }
 
 Type _typeOf<T>() => T; // TODO(kasper): Replace with public version once realm_common contains all
 
 /// @nodoc
 class Mapping<T> {
-  const Mapping();
+  const Mapping({this.indexable = false});
```

Comment:
Are you rebased correctly?

Reply:
No, because you keep force pushing the branch 😅